src/librustdoc/html/static/rustdoc.css

```diff
@@ -417,21 +416,18 @@ h4 > code, h3 > code, .invisible > code {
 .docblock table {
 	border: 1px solid;
 	margin: .5em 0;
-	border-collapse: collapse;
 	width: 100%;
 }
 
 .docblock table td {
 	padding: .5em;
-	border-top: 1px dashed;
-	border-bottom: 1px dashed;
+	border: 1px solid;
 }
 
 .docblock table th {
 	padding: .5em;
 	text-align: left;
-	border-top: 1px solid;
-	border-bottom: 1px solid;
+	border: 1px solid;
```

Comment:
It would be nice if there still was some way to differentiate between the header and normal rows. How about a 2px line at the bottom of the header? Or to just keep a dashed line between the normal rows?

Reply:
If you want.

Reply:
Super nitpick-y, but I'd probably change the general border color to have less of a contrast with the background, and then keep the current color for the thead's border-bottom

Reply:
I'm not sure playing with colors will render well... I'll try but I don't much expectations from it.

Reply:
+1 for thickening the border below the header. I can go either way about changing the border color.